packages/babel-make-styles/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["__fixtures__/**/code.ts", "src"],
```

Comment:
this violates standardised build outputs when builds are executed with `tsc`. While we will loose IDE checking those non production files are being checked on CI. 

This issue will be properly resolved via https://github.com/microsoft/fluentui/pull/19044